site/packages/aa-core/smart-account-client/actions/buildUserOperationFromTx.md

```diff
@@ -14,7 +14,7 @@ head:
 
 # buildUserOperationFromTx
 
-Converts a traditional Ethereum transaction and builds an _unsigned_ `UserOperation` (UO) struct with the all of the middleware run on it through the middleware pipeline.
+A Promise containing the _unsigned_ UO struct converted from the input transaction with all the middleware run on the resulting UO
```

Comment:
This sounds like it describes the return type.. but this section is the description of the method. I think it was ok before, but with a fix:
 ```suggestion
Converts a traditional Ethereum transaction and builds an _unsigned_ `UserOperation` (UO) struct with all of the middleware run on it through the middleware pipeline.
```